features/abbr.yml

```diff
@@ -2,5 +2,7 @@ name: <abbr>
 description: The `<abbr>` HTML element represents an abbreviation or acronym.
 spec: https://html.spec.whatwg.org/multipage/text-level-semantics.html#the-abbr-element
 group: html-elements
+status:
+  compute_from: html.elements.abbr
 compat_features:
   - html.elements.abbr
```

Comment:
This is just to demonstrate the error. See CI logs. I'll of course remove this before merging. 😄